pycronofy/tests/test_real_time_scheduling.py

```diff
@@ -102,5 +105,11 @@ def request_callback(request):
         'hours': 4
     }
 
-    result = client.real_time_scheduling(availability, oauth, TEST_EVENT, [], minimum_notice)
+    callback_url = 'http://www.example.com/callback'
+
+    redirect_urls = {
+        'completed_url': 'http://www.example.com/completed'
+    }
+
+    result = client.real_time_scheduling(availability, oauth, TEST_EVENT, [], minimum_notice, callback_url=callback_url, redirect_urls=redirect_urls)
```

Comment:
REAL_TIME_SCHEDULING_RESPONSE looks different from what is described in this API reference :- 

`
{
  "real_time_scheduling": {
    "real_time_scheduling_id": "sch_4353945880944395",
    "url": "{REAL_TIME_SCHEDULING_URL}"
  }
}
`

Reply:
You're correct, thanks for raising. It doesn't change the behaviour of the code under test, since the method simply passes the deserialized response back to the caller, but I've updated this so that it's a better match for reality